flab_bada/utils/bcrypt.py

```diff
@@ -59,6 +60,23 @@ def create_access_token(data: dict, expires_delta: timedelta | None = None):
     return encoded_jwt
 
 
+def create_confirm_token(data: dict, expires_delta: timedelta | None = None):
+    """
+        토큰 생성
+    Args:
+         data:
+         expires_delta:
+    """
+    to_encode = data.copy()
+    if expires_delta:
+        expire = datetime.utcnow() + expires_delta
+    else:
+        expire = datetime.utcnow() + timedelta(minutes=15)
```

Comment:
```suggestion
def create_confirm_token(data: dict, expires_delta: timedelta = timedelta(minutes=15)):
    to_encode = data.copy()
    expire = datetime.utcnow() + expires_delta
```
default 값을 활용해도 좋겠습니다.

Reply:
넵 감사합니다.